packages/web/src/scss/components/Tab/index.html

```diff
@@ -0,0 +1,43 @@
+{{#> layout }}
+  <h2 class="typography-heading-medium-text mt-1100 mb-800">Tab</h2>
+  <ul class="Tab" role="tablist">
+    <li class="Tab__item">
+      <a
+        href="#pane1"
+        class="Tab__link Tab__link--selected"
```

Comment:
I will need for JS support that CSS must also support some neutral class name like `is-active` for both item and pane to handle switching of tabs correctly and without dependency on the exact class name which could be prefixed.

In JS plugin exact class names must be omitted to guarantee interoperability.

Reply:
@crishpeen Please, checkout this commit https://github.com/lmc-eu/spirit-design-system/pull/347/commits/bbe3818370a28d6ca6f66f8b5416d4f0036bb4c8

Reply:
I would use `is-selected` as it corresponds with the aria attribute `aria-selected="true"`, but you can change try to change my mind :) 

For now, I changed it to `is-selected`.

Reply:
I do not care. I will just change one variable :-) Thanks.